src/core/public/notifications/toasts/toasts_service.mock.ts

```diff
@@ -16,20 +16,29 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { ToastsSetup } from './toasts_start';
+import { ToastsApi } from './toasts_api';
 
-const createSetupContractMock = () => {
-  const setupContract: jest.Mocked<PublicMethodsOf<ToastsSetup>> = {
+const createToastsApiMock = () => {
+  const api: jest.Mocked<PublicMethodsOf<ToastsApi>> = {
     get$: jest.fn(),
     add: jest.fn(),
     remove: jest.fn(),
     addSuccess: jest.fn(),
     addWarning: jest.fn(),
     addDanger: jest.fn(),
   };
-  return setupContract;
+  return api;
+};
+
+const createSetupContractMock = () => {
```

Comment:
These can be specified by reference to avoid the extra function call and allocation:

```js
const createSetupContractMock = createToastsApiMock;
const createStartContractMock = createToastsApiMock;
```